ember-cli/2017-11/november-16.md

```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-11-16
+
+## Attendees
+
+## Agenda
```

Comment:
Post agenda items as comments here...

Reply:
From @locks: 

* RFC unification RFC

Reply:
eslint configuration setup: https://github.com/ember-cli/ember-cli/pull/7443

Reply:
oops, and thank you

Reply:
ember-cli@2.17.0 release (#soon)

* need blog post
* releases (final beta?)

Reply:
Testing in CI on OSX: https://github.com/ember-cli/ember-cli/pull/7440